build/cocos2d_tests.xcodeproj/xcshareddata/xcschemes/cpp-tests Mac.xcscheme

```diff
@@ -49,6 +49,7 @@
       useCustomWorkingDirectory = "NO"
       ignoresPersistentStateOnLaunch = "NO"
       debugDocumentVersioning = "YES"
+      enableAddressSanitizer = "YES"
```

Comment:
Ah, good catch. But yes, I guess it should be enabled. It allows us to catch bugs like stackoverflows.